### ISSUE TITLE
CentOS 8 Support

### ISSUE THREAD

**Issue by linuxliste** [YanChii/ansible-role-postgres-ha#27]
Hi,
Could you support CentOS 8?
Thanks.

**Comment by YanChii** [comment]
Yup. Ansible lib preparation is already done. I'll let you know when it's finished.

**Comment by lentferj** [comment]
Any news on this issue?

**Comment by YanChii** [comment]
Unfortunately not much yet. I'll try to find some time to finish it and I'll let you know.

**Comment by MaxVolum3** [comment]
Hi, any news on this one? Sorry for yet another ping.

**Comment by YanChii** [comment]
Hi.
Thank you for letting me know you're still interested. I'm increasing priority of this task.
Jan
